infrastructure/src/main/java/org/corfudb/infrastructure/AbstractServer.java

```diff
@@ -68,6 +100,33 @@ public final void handleMessage(CorfuMsg msg, ChannelHandlerContext ctx, IServer
         processRequest(msg, ctx, r);
     }
 
+    /**
+     * Handle a incoming request message.
+     *
+     * @param req An incoming request message.
+     * @param ctx The channel handler context.
+     * @param r   The router that took in the request message.
+     */
+    public final void handleRequest(RequestMsg req, ChannelHandlerContext ctx, IServerRouter r) {
```

Comment:
Can you keep the name consistent with https://github.com/CorfuDB/CorfuDB/pull/2812/files#diff-b8f48a536562d8a1808b637fc91cba3ab703038172f68c7eaf35eac773e83f0dR89 

You can overload the method.

Reply:
Done.